### PR TITLE
Agreement for meeting format

### PR DESCRIPTION
**Pull request by jshinm**
Update for the issue #597

### CODE REVIEW

**Comment by jshinm** [comment]
@tathey1 @PSSF23 Jovo asked me to add a section for meeting format based on his recent presentation. Could I please get your feedback?

**Review by tathey1 — COMMENTED** [review]
Can you shorten this to two sentences? Our agreements page is getting quite long in my opinion.

**Review by PSSF23 — COMMENTED** [review]
First, why not put the change in Jovo's PR (#592)? I think he's modifying the same file so conflicts will arise once one of your PRs merged.

**Review by PSSF23 — COMMENTED** [review]
Also, as formalism means: "excessive adherence to prescribed forms," we should reduce the paragraph's size.

**Comment by jshinm** [comment]
Thanks guys! I re-worded the the word in question, and moved the change to Jovo's PR.
